aspnetcore/fundamentals/host/generic-host.md

```diff
@@ -71,6 +71,21 @@ Example `HostBuilder` configuration using `ConfigureHostConfiguration`:
 
 Extension methods are called on the `IHostBuilder` implementation to configure the content root and the environment.
 
+#### Application Key (Name)
+
+The [IHostingEnvironment.ApplicationName](/dotnet/api/microsoft.extensions.hosting.ihostingenvironment.applicationname) property is set from host configuration during host construction. To set the value explicitly, use the [HostDefaults.ApplicationKey](/dotnet/api/microsoft.extensions.hosting.hostdefaults.applicationkey):
+
+**Key**: applicationName  
```

Comment:
Should applicationName be code-fenced?

Reply:
No, we're not doing that because it renders *too blocky-ish* in the rendering of the content. It goes back to a decision that Tom and I discussed ages ago.

Reply:
Of course, Sir Tom is gone now. Gone but in a good place anyway. It's back up to @Rick-Anderson. I still vote for *not blocky* in these groupings.

Reply:
We can revisit later. Rick's on vacation today, and I wouldn't want a minor detail like this to block merging.